server/sonar-server/src/main/java/org/sonar/server/db/migrations/MigrationStepModule.java

```diff
@@ -80,9 +79,6 @@
   @Override
   protected void configureModule() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Protected method 'configureModule(...)' is never used. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedProtectedMethod)